pkgs/top-level/release.nix

```diff
@@ -339,7 +342,7 @@ let
       idrisPackages = packagePlatforms pkgs.idrisPackages;
       agdaPackages = packagePlatforms pkgs.agdaPackages;
 
-      pkgsLLVM.stdenv = [ "x86_64-linux" "aarch64-linux" ];
+      pkgsLLVM = packagePlatformsExclude [ "aarch64-darwin" "x86_64-darwin" ] pkgs.pkgsLLVM;
```

Comment:
Wouldn't this be super expensive on Hydra because it builds _all_ packages for LLVM?

Reply:
It can be, my hope was LLVM to be cached on Hydra at some point. Maybe we at least do CI eval and disable building on Hydra. I'm just not sure how to do that.

Reply:
We have to first scale our build infra to before we can accept new architecture. Also evaluation in hydra is too slow, also I don't know about the latest developments with the new hydra machine.

Reply:
Yeah, that's why I'm thinking of just disabling it for Hydra. Do we have an existing way to do so?

Reply:
`meta.hydraPlatforms = [ ];` is one way, I believe.

Reply:
Yes, that's only on package level.

Reply:
I could make changes to `mkDerivation` that changes that in pkgsLLVM.

Reply:
Yeah, I agree. There should be like an environment attribute which describes how nixpkgs is being used. It could define whether it's in the normal setup, Hydra, or like an eval only setup.

Reply:
Feels like our allowUnfree and friends.

Reply:
Oh lol, I could do `allowBuildingSubpackages` and add that in. If it's off then it masks all `pkgs*` to have Hydra platforms to be empty.

Reply:
I think the behavior I just added in should do the job.